package.json

```diff
@@ -4,7 +4,8 @@
   "description": "In this project, I built a simple HTML Javascript Todo List.",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --config webpack.config.js"
+    "build": "webpack --config webpack.config.js",
+    "deploy": "gh-pages -d dist"
```

Comment:
- [ ] [Optional] Please add `"start": "webpack serve --open",` to scripts to make the dev server run as expected.